### PR TITLE
convert sorted appsdb files list to set

### PR DESCRIPTION
**Pull request by misaka**
Fixes breakage in backup & restore commands.

See issues #1410 & #1408

### CODE REVIEW

File: mackup/appsdb.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def get_files(self, name):
         Returns:
             set of str.
         """
-        return sorted(self.apps[name]['configuration_files'])
+        return set(sorted(self.apps[name]['configuration_files']))
```
Comment (lra) [inline mackup/appsdb.py]: The problem is that a set is not sorted by definition, so we'd rather remove the `sorted` until we use sorted set =/

Comment (misaka) [reply mackup/appsdb.py]: Make sense. 👍

**Comment by lra** [comment]
Thanks for the PR btw, I just did a fix on my side as it's kind of urgent and I did not know if you'd be online. Sorry for takeover!

**Comment by misaka** [comment]
np, just glad to see it fixed. Thanks for the quick response!

**Comment by robinvalk** [comment]
Any eta on a new homebrew version?

**Comment by lra** [comment]
whenever someone does a PR.

**Comment by robinvalk** [comment]
Ah now I got an update! Yesterday there was no new version available via homebrew. Thanks!!

**Pull request closed without merging**